dotnet/src/Functions/Functions.OpenApi/Extensions/OpenApiKernelExtensions.cs

```diff
@@ -200,6 +201,18 @@ public static class OpenApiKernelExtensions
 
     #region private
 
+    /// <summary>The metadata property bag key to use when storing the id of an operation.</summary>
+    private const string OperationExtensionsIdKey = "id";
+
+    /// <summary>The metadata property bag key to use when storing the path of an operation.</summary>
+    private const string OperationExtensionsPathKey = "path";
+
+    /// <summary>The metadata property bag key to use when storing the method of an operation.</summary>
+    private const string OperationExtensionsMethodKey = "method";
+
+    /// <summary>The metadata property bag key to use for the list of extension values provided in the swagger file at the operation level.</summary>
+    private const string OperationExtensionsMetadataKey = "operation-extensions";
```

Comment:
Looks like `operation-extensions` key is used in example as well. Should we consider making this const public, so users will be able to re-use it? If we decide to convert them to public, we can think if we should keep them in `Extensions` class or more generic place.

Reply:
I'm looking into a few options to make these more easily accessible, and will get back on it later.